pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -91,41 +99,49 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-react-app": "^9.1.2",
     "chai": "^4.2.0",
-    "classnames": "^2.2.6",
+    "classnames": "2.3.2",
     "connected-react-router": "^6.8.0",
+    "copy-to-clipboard": "3.3.1",
+    "css-loader": "3.6.0",
     "d3-array": "^2.8.0",
     "d3-format": "^2.0.0",
     "d3-scale": "^3.2.3",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "esbuild": "0.14.43",
     "esbuild-loader": "^2.19.0",
-    "eslint": "^6.8.0",
+    "eslint": "7.32.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^2.3.0",
     "fetch-mock": "^9.10.7",
     "file-loader": "^6.0.0",
     "highlight.js": "^10.2.0",
+    "history": "4.10.1",
     "http-proxy-middleware": "^1.0.5",
     "identity-obj-proxy": "^3.0.0",
+    "immer": "^9.0.3",
     "jest": "^27.5.1",
     "jest-canvas-mock": "^2.4.0",
     "jest-cli": "^27.5.1",
     "jest-environment-enzyme": "^7.1.2",
     "jest-enzyme": "^7.1.2",
     "jest-fetch-mock": "^3.0.3",
+    "jest-junit": "^16.0.0",
```

Comment:
Looks like this is a net-new dependency (`yarn why jest-junit` reports we don't have it in `master`), so it's probably safe to remove this.